packages/create-cloudflare/templates/hello-world-durable-object/py/src/entry.py

```diff
@@ -50,17 +50,15 @@ class Default(WorkerEntrypoint):
     * @returns {Promise<Response>} The response to be sent back to the client
     """
     async def fetch(self, request):
-        # Create a `DurableObjectId` for an instance of the `MyDurableObject`
-        # class named "foo". Requests from all Workers to the instance named
-        # "foo" will go to a single globally unique Durable Object instance.
-        id = self.env.MY_DURABLE_OBJECT.idFromName("foo")
+		# Create a stub to open a communication channel with the Durable Object
+		# instance named "foo".
+		#
+		# Requests from all Workers to the Durable Object instance named "foo"
+		# will go to a single remote Durable Object instance.
+        stub = self.env.MY_DURABLE_OBJECT.getByName("foo")
 
-        # Create a stub to open a communication channel with the Durable
-        # Object instance.
-        stub = self.env.MY_DURABLE_OBJECT.get(id)
-
-        # Call the `say_hello()` RPC method on the stub to invoke the method on
-        # the remote Durable Object instance
+		# Call the `sayHello()` RPC method on the stub to invoke the method on
+		# the remote Durable Object instance.
         greeting = await stub.say_hello("world")
```

Comment:
as my other comment, here we still have `say_hello`, so this name should be updated I believe